lukefi/metsi/domain/sim_ops.py

```diff
@@ -24,6 +24,7 @@
 
 __all__ = ['grow_metsi',
            'grow_acta',
+           'grow_acta_vectorized',
```

Comment:
As the project now move towards the SoA type data model architeture as main data architect. We could consider making a own module for the vectorized operations. Eg. `sim_vec_ops.py`?